src/com/layout/container/container.js

```diff
@@ -0,0 +1,15 @@
+import {h, Component} 				from 'preact/preact';
+
+export default class Container extends Component {
+	constructor( props ) {
+		super(props);
+	}
+
+	render( props ) {
+		return (
+			<div {...props} class={["container", props.class]}>
```

Comment:
`["container", props.class]` should be `cN(props.class, "container")` (`[]` is deprecated).